src/com/google/javascript/jscomp/ConvertChunksToESModules.java

```diff
@@ -209,8 +210,15 @@ private void addImportStatements() {
                   getChunkName(exportingChunk)));
```

Comment:
Error Prone is complaining about line 203's reference to static method `relativePath` via `this`.
Is there some reason not to drop the `this.`?

Reply:
Dropping the `this.` reference should be just fine.